### DISCUSSION TITLE
v2.5.1

### DISCUSSION THREAD

**Discussion by lukevella** [lukevella/rallly#587] [Announcements]
This patch enables Vietnamese language support and prevents posthog from trying to initialize itself in environments where it is not configured to run.

## What's Changed
* 🌐 Update translations by @lukevella in https://github.com/lukevella/rallly/pull/579
* 🌐 Update translations by @lukevella in https://github.com/lukevella/rallly/pull/580
* 🇻🇳 Add Vietnamese Language by @lukevella in https://github.com/lukevella/rallly/pull/581
* 🌐 Update translations by @lukevella in https://github.com/lukevella/rallly/pull/582
* 🌐 Update translations by @lukevella in https://github.com/lukevella/rallly/pull/585
* 📈 Conditionally initialize posthog by @lukevella in https://github.com/lukevella/rallly/pull/586

**Full Changelog**: https://github.com/lukevella/rallly/compare/v2.5.0...v2.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/lukevella/rallly/releases/tag/v2.5.1'>v2.5.1</a>.</em>

**Comment by p-jones** [comment]
Is the version number exposed in the UI anywhere? I feel like an idiot but I cannot see it. Thank you for writing this excellent software.

**Reply by lukevella** [reply]
Not at the moment, but I've been meaning to do it. Will try and get that done soon.
